### ISSUE TITLE
Rename lucia adder to auth

### ISSUE THREAD

**Issue by benmccann** [sveltejs/cli#247]
Folks are getting really confused and thinking it's using a deprecated library

**Comment by Rich-Harris** [comment]
It does feel a bit 'one true auth solution' if we do that, and I'm not sure that it is — I can't log in with google or github or biometrics or a passkey or whatever, there's no bot detection or email verification or password resets or the many other things that go into authentication, and we don't have anything to say about authorization. And it's tied to one specific ORM.

I'm not saying `lucia` is a name that we should keep, but I'm a little wary of people's expectations around `auth`. I can imagine Laravel developers saying 'aww, that's cute'.

**Comment by manuel3108** [comment]
Maybe `basic-auth` or something

**Comment by benmccann** [comment]
I think we can add those things over time. We need to start somewhere

`basic-auth` feels weird to me because then as we add features, at what point do we remove `basic-` from the name?

**Comment by Conduitry** [comment]
'Basic auth' very much has an existing precise meaning already, which I really don't think we should try to go against.

**Comment by AdrianGonz97** [comment]
How about `lucia-guide` or `lucia-v4`? We could also add `lucia` as an alias so that people can still write `npx sv add lucia`

**Comment by benmccann** [comment]
yeah, someone had suggested `lucia-guide` on discord - though someone else then responded that it sounds like you're setting up documentation or something along those lines

I kind of want to send a PR for a feature that's not in the Lucia guide just so we can stop calling it anything related to Lucia :laughing:

**Comment by AdrianGonz97** [comment]
`lucia-inspired` then?

gives us leeway to change things in the future while also referencing where it loosely came from

**Comment by benmccann** [comment]
sounds fine to me

**Comment by Rich-Harris** [comment]
`email-password-auth`? It's not like Lucia is doing anything particularly unusual

**Comment by Rich-Harris** [comment]
(or maybe not that exact thing but you get the idea — something that says what it does)

**Comment by AdrianGonz97** [comment]
i think at that point we should just call it `auth` so that it prompts a multiselect for the different auth methods like: `email-password`, `oauth`, `passkey`, etc

```
┌  Welcome to the Svelte CLI! (v0.5.11)
│
◇  What would you like to add to your project?
│  auth
│
◆  Which auth methods would you like to include?
│  ◻ email-password
│  ◻ oauth
│  ◻ passkey
│  ...
└

**Comment by AdrianGonz97** [comment]
also, yes, it would look a bit silly _for the time being_ since there would only be a single auth method available:
```
┌  Welcome to the Svelte CLI! (v0.5.11)
│
◇  What would you like to add to your project?
│  auth
│
◆  Which auth methods would you like to include?
│  ◻ email-password
└
```

**Comment by MonsalvoGeoffrey** [comment]
![image](https://github.com/user-attachments/assets/33690407-8797-4566-9ab8-6e8b0f564b88)
Since each adder has text that is normally used to link to the library's documentation, I feel like this text is a better place to solve this problem that the title

Having it titled `auth` and the text beside it reading `Following lucia's guideline (https://lucia-auth.com)` would make it perfectly clear
Sure, every other add-on only have a link, but I don't think it HAVE to be consitent, if lucia's an edge case, then it get the treatment of an edge case
